Sources/Core/AWSClientRuntime/HttpContextBuilder+Extension.swift

```diff
@@ -56,6 +52,16 @@ extension HttpContext {
         // setup client to server
         let messageEncoder = AWSClientRuntime.AWSEventStream.AWSMessageEncoder()
         let messageSigner = AWSClientRuntime.AWSEventStream.AWSMessageSigner(encoder: messageEncoder) {
+            guard let authScheme = self.getSelectedAuthScheme() else {
+                throw ClientError.authError(
+                    "Signer for event stream could not be loaded because auth scheme was not configured."
+                )
+            }
+            guard let signer = authScheme.signer else {
+                throw ClientError.authError("Signer was not configured for the selected auth scheme.")
+            }
+            return signer
+        } signingConfig: {
             try await self.makeEventStreamSigningConfig()
         } requestSignature: {
             self.getRequestSignature()
```

Comment:
this should be as follows

```
signingConfig: { something },
requestSignature: { something },
```

since these are short they should be scoped and implemented all on one line with a comma after the closing brackets and then onto the next

Reply:
Changed formatting to be easier to read, following given suggestion!